libs/R-TypeLogic/EntityManipulation/CreateEntitiesFunctions/CreateEnemy.cpp

```diff
@@ -25,17 +27,14 @@ namespace ecs
             .addComponent<DamageRadius>(damageRadius)
             .addComponent<Collidable>()
             .addComponent<Velocity>(multiplierAbscissa, multiplierOrdinate)
-            .addComponent<Enemy>(type);
-
+            .addComponent<Enemy>(type)
```

Comment:
Useless in mutliplayer mod, did you envisage make a condition with an intern value of the World ?

Reply:
Enemy type is useless in multiplayer mod ???